go-controller/pkg/ovn/master.go

```diff
@@ -662,9 +662,13 @@ func (oc *DefaultNetworkController) syncChassis(nodes []*kapi.Node) error {
 		return fmt.Errorf("failed to get chassis private list: %v", err)
 	}
 
-	templateVarList, err := libovsdbops.ListTemplateVar(oc.nbClient)
-	if err != nil {
-		return fmt.Errorf("failed to get template var list: error: %v", err)
+	templateVarList := []*nbdb.ChassisTemplateVar{}
+
+	if oc.svcTemplateSupport {
+		templateVarList, err = libovsdbops.ListTemplateVar(oc.nbClient)
+		if err != nil {
+			return fmt.Errorf("failed to get template var list: error: %v", err)
```

Comment:
use %w instead of %v to wrap err into something that we can check against.

Reply:
Done.